pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
+
+`stack build yesod-bin cabal-install --install-ghc`
+
+- Start development server:
+
+`stack -- exec yesod devel`
```

Comment:
Got it. Please add a second line the in the main description that 'All commands are invoked through the `stack` package manager'. This will automatically refer users to the tldr stack page whenever its created.